src/Avalonia.Themes.Fluent/Controls/ManagedFileChooser.xaml

```diff
@@ -0,0 +1,324 @@
+                                        <TextBlock Grid.Column="2" Text="{Binding DisplayName}"/>
+                                        <TextBlock Grid.Column="4" Text="{Binding Modified}" />
+                                        <TextBlock Grid.Column="6" Text="{Binding Type}" />
+                                        <TextBlock Grid.Column="8">
+                                            <TextBlock.Text>
+                                                <Binding Path="Size">
+                                                    <Binding.Converter>
+                                                        <dialogs:FileSizeStringConverter/>
+                                                    </Binding.Converter>
+                                                </Binding>
+                                            </TextBlock.Text>
+                                        </TextBlock>
+                                    </Grid>
+                                </DataTemplate>
+                            </ListBox.ItemTemplate>
+                        </ListBox>
+                    </DockPanel>
+                </DockPanel>
+            </ControlTemplate>
+        </Setter>
+    </Style>
+
+
+    <Style Selector="dialogs|ManagedFileChooser /template/ ListBox#QuickLinks">
+        <Setter Property="Margin" Value="0"/>
+        <Setter Property="Background" Value="{DynamicResource SystemControlBackgroundChromeMediumBrush}"/>
+        <Setter Property="BorderThickness" Value="0"/>
+        <Setter Property="Width" Value="250"/>
+        <Setter Property="Padding" Value="0,20"/>
+        <Setter Property="Template">
+            <ControlTemplate>
+                <Border Name="border" BoxShadow="inset -6 0 3 -3 #20000000" Background="{TemplateBinding Background}" BorderBrush="{TemplateBinding BorderBrush}" BorderThickness="{TemplateBinding BorderThickness}">
+                    <ScrollViewer Name="PART_ScrollViewer" HorizontalScrollBarVisibility="{TemplateBinding (ScrollViewer.HorizontalScrollBarVisibility)}" VerticalScrollBarVisibility="{TemplateBinding (ScrollViewer.VerticalScrollBarVisibility)}">
+                        <ItemsPresenter Name="PART_ItemsPresenter"
+                                        Items="{TemplateBinding Items}"
+                                        ItemsPanel="{TemplateBinding ItemsPanel}"
+                                        ItemTemplate="{TemplateBinding ItemTemplate}"
+                                        Margin="{TemplateBinding Padding}"
+                                        VirtualizationMode="{TemplateBinding VirtualizationMode}"/>
+                    </ScrollViewer>
+                </Border>
+            </ControlTemplate>
+        </Setter>
+    </Style>
+    <Style Selector="dialogs|ManagedFileChooser /template/ ListBox#QuickLinks ListBoxItem">
+        <Setter Property="Height" Value="32"/>
+        <Setter Property="Padding" Value="34,6"/>
```

Comment:
I think this padding can be a bit smaller, like 30 instead of 34 or so. Also ListBox width from 250 to ~240.

Reply:
Done and done :+1: